lambda/variables.tf

```diff
@@ -132,6 +132,11 @@ variable "keycloak_reporting_client_secret" {
   default     = ""
 }
 
+variable "reporting_slack_channel_id" {
+  description = "Slack channel id where TDR reports will be sent"
+  default     = ""
```

Comment:
The default value is fine (""), but just a note for when this is released, I think the encryption will fail on an empty string so will need to release in the correct way.